homework3/task03/tests/test_filter.py

```diff
@@ -0,0 +1,39 @@
+from typing import Any, Dict, List
+
+import pytest
+from project_filter.filter import Filter, make_filter
+
+sample_data = [
+    {
+        "name": "Bill",
+        "last_name": "Gilbert",
+        "occupation": "was here",
+        "type": "person",
+    },
+    {
+        "is_dead": True,
+        "kind": "parrot",
+        "type": "bird",
+        "name": "polly",
+    },
+]
+
+
+@pytest.mark.parametrize(
+    ("data", "expected_result"),
+    [
+        ({"name": "polly"}, [sample_data[1]]),
+        ({"name": "Bill", "occupation": "was here"}, [sample_data[0]]),
+        ({"name": "Iuliia"}, []),
+    ],
+)
+def test_make_filter(data: Dict[Any, Any], expected_result: List[Any]):
```

Comment:
Why do you need sample_data in the test with the name `test_make_filter`?

Reply:
Do you imply this test should be renamed? Fixed